src/main/java/vazkii/botania/common/item/relic/ItemLokiRing.java

```diff
@@ -235,6 +239,11 @@ public static String getLokiBreakingModeText(ItemStack stack){
 				EnumChatFormatting.RED + StatCollector.translateToLocal("botaniamisc.lokiOff"));
 	}
 
+	public static String getLokiCearText(ItemStack stack){
+		return EnumChatFormatting.GOLD + StatCollector.translateToLocal("botaniamisc.lokiClear") ;
+				
+	}
```

Comment:
Various bits of excess whitespace, before the semicolon and at the end of the function.